wazuh/wazuh-kibana-app
# Change macOS register agent installation command

### Description
Hi team, 
this PR changes the agent installation command for macOS.

### Issues Resolved
Closes #5466 

Related to #5481 

### Evidence
![Screenshot from 2023-05-24 12-07-01](https://github.com/wazuh/wazuh-kibana-app/assets/9343732/1c430144-f1a7-4801-9891-c8e623dd813b)

### Test cases

- Go to Agents
- Click on "Deploy new agent"
- On step 1 select `macOS`
  - With Password
    - Check the password is shown by default with the correct value
    - Fill in the rest of the configuration and check the double quotes `"` are escaped `\"`
    - Check the environment variables are separated by `&&`
   - Without password
     - When all inputs are empty the `echo` section of the script should not appear
     - Fill in the rest of the configuration and check the double quotes `"` are escaped `\"`
     - Check the environment variables are separated by `&&`

### Check List
